subprojects/core-api/src/main/java/org/gradle/api/file/DuplicatesStrategy.java

```diff
@@ -40,23 +40,23 @@
     EXCLUDE,
 
     /**
-     * Do not attempt to prevent duplicates, but log a warning message when multiple items 
+     * Do not attempt to prevent duplicates, but log a warning message when multiple items
      * are to be created at the same path.
      * <p>
      * This behaves exactly as INCLUDE otherwise.
      */
     WARN,
 
     /**
+     * The default strategy since 6.0
      * Throw a {@link DuplicateFileCopyingException} when subsequent items are to be created at the same path.
-     * <p> 
+     * <p>
      * Use this strategy when duplicates are an error condition that should cause the build to fail.
      */
     FAIL,
```

Comment:
Maybe this should only be the default strategy for archives. I don't see why the duplicate strategy would be a problem for copying/synching.

Reply:
Is it even possible to have duplicates for non-archives?

Reply:
Not on the disc, though we detect the duplicates nontheless. This is why the sync action fails in the native test: https://github.com/gradle/gradle/blob/1883a66c73d52315e3eb10f54a0451ebe3c92e17/subprojects/platform-native/src/main/java/org/gradle/nativeplatform/tasks/InstallExecutable.java#L263-L267

Reply:
How about we change the default only on `AbstractArchiveTask`?

Reply:
Yeah, that's what I attempted based on your comment that the default for copy shouldn't change.